doc/src/quickstart/index.rst

```diff
@@ -297,9 +269,9 @@ Next, we'll optimize and technology map our circuit using ABC, providing the opt
 
 We'll use the following, simple ABC commands::
 
```

Comment:
parmys works a little differently than Odin-II; it shoul have already optimized and technology mapped the design.  I suggest you take a look at the .blif file that comes out of parmys and see if it is mostly LUTs (.names), with some having up to 6 inputs.  If so, it has been tech-mapped. We can still show people how to run abc, but should modify the text to say parmys already called abc internally to optimize the design, but if we'd like to try some further optimization or technology mapping options we can also directly invoke abc on the parmys output.